.superpowers/brainstorm/21244-1779445273/content/form-display.html

```
@@ -0,0 +1,175 @@
+<h2>人工介入表单 — 展示方式</h2>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)